checker/jtreg/nullness/nulltypetest/NullTypeTest.out

```diff
@@ -0,0 +1,10 @@
+NullTypeTest.java:11:17: compiler.err.proc.messager: [assignment.type.incompatible] incompatible types in assignment.
+found   : null (NullType)
+required: @NonNull Object
+NullTypeTest.java:13:18: compiler.err.proc.messager: [type.argument.type.incompatible] incompatible type argument for type parameter T of A.
+found   : @NonNull NullTypeTest.@NonNull A<@NonNull Object>
+required: [extends @Nullable Object super null (NullType)]
```

Comment:
For the lower bound we shouldn't output `null`. So maybe the special handling for `AnnotatedNullType` needs more thought.